datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -375,7 +407,10 @@ def process_metric(self, metric, scraper_config, metric_transformers=None):
         # If targeted metric, store labels
         self._store_labels(metric, scraper_config)
 
+        self._send_telemetry_counter(self.TELEMETRY_COUNTER_METRICS_PROCESS_COUNT, 1, scraper_config)
+
         if metric.name in scraper_config['ignore_metrics']:
+            self._send_telemetry_counter(self.TELEMETRY_COUNTER_METRICS_IGNORE_COUNT, 1, scraper_config)
```

Comment:
So ignore is a subset of process? I would expect metrics to either be processed or ignored. Or maybe processed is higher level, and we need to subdivide between ignored and considered (that's a shitty name but you get the idea).

Reply:
here my initial idea was:
- TELEMETRY_COUNTER_METRICS_PROCESS_COUNT => the metric reach the `process()` function
- TELEMETRY_COUNTER_METRICS_IGNORE_COUNT => the metrics reach the `process()` function but is ignored

Reply:
> And the considered metric can be calculated at query time. that why I didn't added it.

You're thinking about the metric from the aggregator? I don't think it has the instance tags so if we want the same level of details as the ones you create which come with `_metric_tags` and `custom_tags` we should create a new metric here.

```
TELEMETRY_COUNTER_METRICS_PROCESS_COUNT => the metric reach the process() function
TELEMETRY_COUNTER_METRICS_IGNORE_COUNT => the metrics reach the process() function but is ignored```

Gotcha. Imho since the ignored metrics have a fast path out of process they don't qualify as processed metrics.